src/renderer/components/catalog/PublicationAddButton.tsx

```diff
@@ -44,18 +40,23 @@ export class PublicationAddButton extends React.Component<Props> {
         );
     }
 
-    private importFile(event: any) {
+    private importFile(event: TChangeEvent) {
         const files = event.target.files;
         const paths: string[] = [];
 
-        for (const f of files) {
-            paths.push(f.path);
+        // files: FileList doesn't have [symbol.interator()]
+        // tslint:disable-next-line: prefer-for-of
```

Comment:
Suggestion: edit `tsconfig.json`, change `"lib": ["es2017", "es6", "dom"],` to `"lib": ["es2017", "es6", "dom", "dom.iterable"],` (added `"dom.iterable"`), and restore the `for (const f of files)` code.

Reply:
aae238c